### PR TITLE
docs: trim task README maturity claims

### PR DESCRIPTION
**Pull request by liq22**
## Summary

- Replace speculative task README content for `DG`, `CDDG`, `FS`, `GFS`, and `pretrain` with concise documentation of the current code and maintained config surface.
- Remove unsupported SOTA, dataset-count, production-readiness, and unimplemented roadmap claims.
- Document maintained demo paths, registry sources of truth, key config fields, and explicit support boundaries for each task family.

## Why

The previous module READMEs mixed current behavior with research ideas and planned capabilities. That could cause users to treat unimplemented losses, samplers, training stages, and methods as release-supported. The replacement docs distinguish maintained demos from merely registered implementations and direct users to the config registry and inspect tooling.

## Validation evidence

- `python -m scripts.validate_docs` — passed locally.
- `python -m scripts.validate_configs` — passed locally.
- `git diff --check origin/main..HEAD` — passed.
- Maturity-claim keyword scan over all five replacement READMEs returned no matches.

## Maintainer review against current main

- `DG`: base config, maintained demo paths, and `task.loss`/`task.metrics`/optimizer fields match the tracked configs.
- `CDDG`: `target_system_id` and `target_domain_num` match `configs/base/task/cddg.yaml` and the maintained cross-system demo.
- `FS`: maintained demo uses `FS.classification`; registered task variants match `src/task_factory/task_registry.csv`; few-shot fields match `configs/base/task/fewshot.yaml`.
- `GFS`: registered variants, maintained demo, and sampler fields match `configs/base/task/cddg_fewshot.yaml`.
- `pretrain`: registered variants and maintained HSE demo paths match the registry and tracked demo configs.
- The final diff changes only five README files. No runtime, config, registry, tests, or release documents are modified.
- GitHub reports the PR mergeable against current `main`; no unresolved review threads exist. GitHub Actions are not configured for this head.

### CODE REVIEW

**Comment by liq22** [comment]
Local validation after opening the draft PR:

```text
python -m scripts.validate_docs
[OK] Documentation checks passed (111 files scanned).

python -m scripts.validate_configs
[OK] 7/7 configs passed schema validation.

git diff --check origin/main..HEAD
# passed with no output

rg -n 'TODO|SOTA|state-of-the-art|30\\+|20\\+|fully|complete|benchmark-valid|production|NOT IMPLEMENTED|Compatible with all|Support for all|Comprehensive|robust|one-click|一键|革命性|AI 辅助|vibecoding' \
  src/task_factory/task/DG/README.md \
  src/task_factory/task/CDDG/README.md \
  src/task_factory/task/FS/README.md \
  src/task_factory/task/GFS/README.md \
  src/task_factory/task/pretrain/README.md
# no matches

git rev-list --left-right --count origin/main...HEAD
0 1

git merge-tree 5763f8f2bc2ed15d65fd951d2ddcda10692a3651 origin/main HEAD | rg -n '<<<<<<<|=======|>>>>>>>'
# no conflict markers
```

Scope is docs-only: five task README files.
